payment.go

```diff
@@ -31,3 +118,38 @@ func (p Payment) CreditCardExpirationDateString() string {
 	month := p.Faker.IntBetween(12, 30)
 	return fmt.Sprintf("%02d/%02d", day, month)
 }
+
+// Iban returns a fake IBAN for Payment
+func (p Payment) Iban() string {
+	return p.ibanForCountry(p.Faker.RandomStringMapKey(ibanFormats))
+}
+
+// ibanForCountry returns a fake IBAN for a specific country
+// Returns an empty string if the country is not supported
+func (p Payment) ibanForCountry(countryCode string) string {
+	format, ok := ibanFormats[countryCode]
+	if !ok {
+		return ""
+	}
+
+	bban := strings.ToUpper(p.bban(format))
+	checksum := ibanChecksum(countryCode + "00" + bban)
+
+	return countryCode + checksum + bban
+}
+
+// bban generates a fake Basic Bank Account Number (BBAN) based on the format
+// the provided format must be a string only containing the following characters:
+// n: numeric, a: alphabetic, c: alphanumeric
+// which will be replaced by a random number or letter
+func (p Payment) bban(format string) string {
+	format = strings.ReplaceAll(format, "n", "#")
+	format = strings.ReplaceAll(format, "a", "?")
+	s := "?"
+	if rand.Intn(100) <= 50 {
```

Comment:
Please use `p.Faker.IntBetween(min, max)` instead

Reply:
or if your intention is to simply "flip a coin", you can also use `p.Faker.Bool()`